examples/reference_environments/gcs/compose.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+  jupyter:
+    image: gx/gcs_example_jupyter
+    container_name: gx_gcs_example_jupyter
+    build:
+      context: .
+      dockerfile: jupyter.Dockerfile
+    environment:
+      GOOGLE_APPLICATION_CREDENTIALS: /tmp/creds.json
```

Comment:
How about passing through `GCP_PROJECT_NAME` here from the host? (see related comment in jupyter notebook)

Reply:
done!